### DISCUSSION TITLE
Anyone knows heat cost allocators that can be read?

### DISCUSSION THREAD

**Discussion by LBoese** [wmbusmeters/wmbusmeters#1419] [Q&A]
Hi, 
does anyone know a type of heat cost allocator that was successfully read with wmbusmeters? As I have not yet installed any heating cost allocator, I am looking for a manufacturer whose type can be read out. Thank you

**Comment by Tpaydt** [comment]
https://qundis.de/produkte/heizkostenverteiler/
Q caloric 5.5
i have this and works fine with wmbusmeters

**Comment by Gesso242** [comment]
What about Sontex 868? Any suggestions oder drivers?
